tests/UnitTests/DogStatsd/SocketsTest.php

```diff
@@ -26,11 +26,49 @@ static function getPrivate($object, $property) {
         return $reflector->getValue($object);
     }
 
+    private function get_default(&$var, $default=null) {
+      return isset($var) ? $var : $default;
+    }
+
+    public function assertSameTelemetry($expected, $actual, $message="", $params = array())
```

Comment:
nit: I was a bit confused by the name at first, maybe:
```suggestion
    public function assertSameWithTelemetry($expected, $actual, $message="", $params = array())
```

is better?